setup.py

```diff
@@ -280,5 +280,4 @@ def build_extensions(self):
             build_ext=CMakeBuild, install_hook=install_hook, build_hook=build_hook
         ),
         zip_safe=False,
-        use_scm_version={"fallback_version": "unknown-no-.git-directory"},
```

Comment:
Check the `if` as well.

Reply:
Nice catch - totally forgot about our honking hack there.